tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestProject.Fixed.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.Razor" Version="3.2.7" />
+    <PackageReference Include="Microsoft.AspNet.Mvc" Version="3.2.7" />
+    <PackageReference Include="Microsoft.Owin" Version="4.1.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="*\UA*.cs" />
+    <Compile Include="*\UA*.Fixed.cs" />
```

Comment:
Not sure how we get around having classes with the same name in 2 separate files but this was causing problems in the VBproj so I decided to apply it to the CSproj to mitigate any potential future issues.